packages/kit/src/runtime/server/page/csp.spec.js

```diff
@@ -97,20 +107,27 @@ test('skips nonce with unsafe-inline', () => {
 	);
 
 	csp.add_script('');
+	csp.add_style('');
 
-	assert.equal(csp.csp_provider.get_header(), "default-src 'unsafe-inline'");
-	assert.equal(csp.report_only_provider.get_header(), "default-src 'unsafe-inline'; report-uri /");
+	assert.equal(
+		csp.csp_provider.get_header(),
+		"default-src 'unsafe-inline'; script-src 'unsafe-inline'; script-src-elem 'unsafe-inline'; style-src 'unsafe-inline'; style-src-attr 'unsafe-inline'; style-src-elem 'unsafe-inline'"
+	);
+	assert.equal(
+		csp.report_only_provider.get_header(),
+		"default-src 'unsafe-inline'; script-src 'unsafe-inline'; script-src-elem 'unsafe-inline'; style-src 'unsafe-inline'; style-src-attr 'unsafe-inline'; style-src-elem 'unsafe-inline'; report-uri /"
+	);
 });
 
-test('skips nonce in style-src when using unsafe-inline', () => {
```

Comment:
What was the reason for this test being removed / refactored?

Reply:
I'm not sure to be honest, i wrote this a long time ago. I think it was because it was redundant, but i am not certain about this

Reply:
I've added the test back again @eltigerchino! It still passes, so i guess it was because of it being redundant. But it's just better to keep it there i guess 👍